### ISSUE TITLE
Neighbourhoods duplicate of suburbs

### ISSUE THREAD

**Issue by zerebubuth** [tilezen/vector-datasource#744]
tl;dr

Remove our export of the following OSM features in favor of their WOF "neighbourhood" equivalents. 

- borough
- suburb
- quarter

===

Original report:

**What did you see?**

Duplicate labels for Fluntern and Hottingen in Zurich.

**What did you expect to see?**

Only a single label for each name.

**What map location are you having problems with?**

Thanks to @simonpoole for pointing it out on [this Mapillary map](http://www.mapillary.com/map/im/vu3TtIen40ghnSH8QrdZiQ/map). Also visible [in a demo](https://tangrams.github.io/tangram-sandbox/tangram.html?styles/patterns.yaml#18/47.37719/8.55892), but most seem to not show suburb labels.

**Screenshot?**

![image](https://cloud.githubusercontent.com/assets/271360/14779883/209cded6-0ad2-11e6-988b-4d816bc5e113.png)

Tile `places/15/17163/11474` contains:

```json
    {
      "geometry": {
        "type": "Point",
        "coordinates": [
          8.558931,
          47.376725
        ]
      },
      "type": "Feature",
      "properties": {
        "kind": "suburb",
        "scalerank": 10,
        "source": "openstreetmap.org",
        "name:gsw": "Fluentere",
        "id": 525903857,
        "name": "Fluntern"
      }
    },
    {
      "geometry": {
        "type": "Point",
        "coordinates": [
          8.558695,
          47.377594
        ]
      },
      "type": "Feature",
      "properties": {
        "kind": "neighbourhood",
        "name": "Fluntern",
        "area": 741717,
        "max_zoom": 18,
        "kind_tile_rank": 1,
        "source": "whosonfirst.mapzen.com",
        "min_zoom": 13,
        "id": 85799981
      }
    }
```

**Comment by skorasaurus** [comment]
Can confirm this on the cinnabar style in Cleveland, Ohio. Looks like Mapzen needs to determine whether to use only whosonfirst or only osm for suburb and neighbourhood tagged places or conflate the data sources.

**Comment by nvkelso** [comment]
Doh! Good find. 

Background: since [v0.5.0](https://github.com/mapzen/vector-datasource/blob/master/CHANGELOG.md#v050) of Mapzen Vector Tiles we prefer Who's On First features for `neighbourhoods` (we no longer export OSM neighbourhoods at all in the `places` layer).

Reading up on OSM's [`place=suburb`](http://wiki.openstreetmap.org/wiki/Tag:place%3Dsuburb), we should probably always ignore this OSM kind in favor of the WOF `neighbourhood` features, as their definitions overlap.

**Comment by nvkelso** [comment]
Are there other features we should ignore, like `quarter` and/or `borough`?

**Comment by simonpoole** [comment]
Well lets see, best would be to ignore WOF (only 1/2 :-))

Seriously the map tiles shown by mapillary are now even more broken than they were before with great expanses of emptiness where there are thriving towns with 10s of 1000s of inhabitants.

**Comment by nvkelso** [comment]
The benchmark to compare with is the Mapzen house styles. The Mapillary thing needs some serious TLC.

**Comment by nvkelso** [comment]
Confirmed we should remove:

- [ ] borough
- [ ] suburb
- [ ] quarter

**Comment by nvkelso** [comment]
Verified on dev.
